snippets/language-mjml.cson

```diff
@@ -190,6 +190,27 @@
       <mj-carousel-image src="$1" />
     """
 
+  'MJML Wrapper':
+    'prefix': 'wrapper'
+    'body': """
+      <mj-wrapper>\n\t$1\n</mj-wrapper>
+    """
+
+  'MJML Accordion':
+    'prefix': 'accordion'
+    'body': """
+      <mj-accordion>\n\t$1\n</mj-accordion-element>
```

Comment:
It should be `<mj-accordion>\n\t$1\n</mj-accordion>` instead of `<mj-accordion>\n\t$1\n</mj-accordion-element>`